src/core/a-animation.js

```diff
@@ -356,13 +357,14 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
   var from = {};
   var partialSetAttribute;
   var to = {};
-
-  if (attributeSplit.length === 2) {
+  if (attributeSplit.length === 2 && attributeSplit.indexOf('color') === -1) {
```

Comment:
Ok I have something like: 

```
function isColor() {
  if (el.mappings && attributeSplit.length === 1 && !el.components[attribute]){
    return el.components[el.mappings[attribute].split('.')[0]].schema[attribute].type === 'color';
  } else if(attributeSplit.length > 1 && el.components[attributeSplit[0]]) {
    return el.components[attributeSplit[0]].schema[attributeSplit[1]].type === 'color';
  }
  return false;
}
```

There has to be an easier way to access the schema?
The reason I had to do this was to access the correct schema on a single attribute 'color' I had to access the mappings because there was no color schema it was under 'material.color' as that is the default map.